src/main/java/org/session13/homework/librarymanagementsystem/Main.java

```diff
@@ -0,0 +1,99 @@
+package org.session13.homework.librarymanagementsystem;
```

Comment:
Please consider the structure of packages as follows:
src 
domain (this package will contain all the core domain objects/entities of your library system)
- Author
- Book
- Member

services (this will house all service classes which contain business logic for various operations in your library system)
- LibraryServices
- MemberServices

application (this package will contain classes responsible for bootstrapping and running the application)
- Library

in the future we could also have packages like `repository`, `exceptions` and `utilities`